sdk/lib/indexed_db/indexed_db_sources.gni

```diff
@@ -0,0 +1,7 @@
+# Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+# for details. All rights reserved. Use of this source code is governed by a
+# BSD-style license that can be found in the LICENSE file.
+
+indexed_db_sdk_sources = [
+    "dart2js/indexed_db_dart2js.dart",
+]
```

Comment:
Missing the new line here and in the svg_sources.gni below (make sure to run `gn format ...`) on these newly added files.